### DISCUSSION TITLE
Is there a ready-to-use project to transform md files to web?

### DISCUSSION THREAD

**Discussion by flaviotordini** [11ty/eleventy#2962] [Uncategorized]
I would like to process a set of generic Markdown files that were not written for 11ty. Before I go on and try to write my own I would like to know if there is some kind of 11ty-based project that tries to be a "generic" tool for users, instead of being a starter project for devs. Thanks in advance!

**Comment by pdehaan** [comment]
I think I'm confused. How would generic Markdown files be different than Markdown+11ty files?
Yesterday I converted a Google Doc into basic Markdown and used that with Eleventy without needing to do anything special to get it to get it to work with 11ty. Or are you referring to optional front matter, etc?

**Reply by flaviotordini** [reply]
Hi, I'm referring to basic things such as preserving links among documents, copying images, maybe generate a directory index when missing, a basic style, a basic layout.

**Comment by jg3** [comment]
It sounds like what you're describing is pretty much what 11ty does.   All the "build a website in 43 seconds with Eleventy!" tutorials are laid out with the markdown files ("templates" in Eleventy parlance) in a certain folder structure, but where your `.md` files live is the "input" configuration line in the eleventy config file.

Note that Eleventy won't automatically generate a listing of the files for you, but you would likely benefit from taking  one of the several eleventy "starters" which might have some of that done for you.
